devtools-ci-index.yaml

```diff
@@ -898,6 +898,12 @@
             timeout: '30m'
             svc_name: keycloak-server
         - '{ci_project}-{git_repo}':
+            git_organization: almighty
+            git_repo: keycloak
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '20m'
```

Comment:
Maybe one option could be to build it without tests to then repeat it with tests but without `cleaning`.